docs/intro/tutorial.rst

```diff
@@ -323,33 +323,33 @@ use Firefox Developer Tools or some Firefox extensions like Firebug. For more
 information see :ref:`topics-firebug` and :ref:`topics-firefox`.
 
 After inspecting the page source, you'll find that the web site's information
-is inside a ``<ul>`` element, in fact the *second* ``<ul>`` element.
+is inside a ``<div class="title-and-desc">`` element.
 
-So we can select each ``<li>`` element belonging to the site's list with this
+So we can select each ``<div class="title-and-desc">`` element belonging to the site's list with this
 code::
 
-    response.xpath('//ul/li')
+    response.xpath('//div[@class="title-and-desc"]')
 
 And from them, the site's descriptions::
 
-    response.xpath('//ul/li/text()').extract()
+    response.xpath('//div[@class="title-and-desc"]//div[@class="site-descr "]/text()').extract()
 
 The site's titles::
 
-    response.xpath('//ul/li/a/text()').extract()
+    response.xpath('//div[@class="title-and-desc"]//div[@class="site-title"]/text()').extract()
 
 And the site's links::
 
-    response.xpath('//ul/li/a/@href').extract()
+    response.xpath('//div[@class="title-and-desc"]/a/@href').extract()
 
 As we've said before, each ``.xpath()`` call returns a list of selectors, so we can
 concatenate further ``.xpath()`` calls to dig deeper into a node. We are going to use
 that property here, so::
 
-    for sel in response.xpath('//ul/li'):
-        title = sel.xpath('a/text()').extract()
+    for sel in response.xpath('//div[@class="title-and-desc"]'):
+        title = sel.xpath('//div[@class="site-title"]/text()').extract()
```

Comment:
Nice work, @andrewpost!

Just an observation: here we should be using relative XPath expressions such as:
```
title = sel.xpath('.//div[@class="site-title"]/text()').extract()
desc = sel.xpath('.//div[@class="site-descr "]/text()').extract()
```

Or even simpler ones:
```
title = sel.xpath('a/div/text()').extract()
desc = sel.xpath('div/text()').extract()
```

The same observation applies to subsequent occurrences of the same code.